examples/aws.md

```diff
@@ -26,6 +26,10 @@ upstreams:
    autoscaling_group: backend-two-group
    port: 80
    kind: http
+ - name: backend
```

Comment:
I suggest removing this snippet. The example config reflects the setup described in the main readme -- https://github.com/nginxinc/nginx-asg-sync#nginx-plus-configuration , which doesn't include the third upstream,